e2e/Microsoft.Azure.Devices.E2ETests/TwinE2ETests.cs

```diff
@@ -391,6 +391,74 @@ static public void ClassCleanup()
             await _Twin_DeviceSetsReportedPropertyAndServiceReceivesIt(Client.TransportType.Amqp_WebSocket_Only);
         }
 
+        [TestMethod]
+        [TestCategory("Twin-E2E")]
+        public async Task Twin_ServiceDoesNotCreateNullPropertyInCollection_Mqtt()
+        {
+            // GitHub Issue #243
```

Comment:
We don't usually leave comments regarding which GH issue created the test. We keep the numbers when we add //TODO statements that track future work. We remove the TODOs once the work is done such that in the end we leave no issue numbers in the code (these would eventually go stale if we moved GH accounts as we did in the past).

I think the name of the test is sufficient to describe what it does.

Reply:
Removed...